.github/workflows/build-test.yml

```diff
@@ -362,6 +362,10 @@ jobs:
         os: [macos-latest, ubuntu-latest, windows-latest]
     name: Unit tests
     steps:
+      - name: disable git autocrlf
+        run: |
+          git config --global core.autocrlf false
```

Comment:
the test files are read verbatim, which means it's problematic if `git` is allowed to insert `\r` when checking out files

Reply:
Would you consider putting this comment into the YAML file itself?

Reply:
good point